requirements.txt

```diff
@@ -1,13 +1,18 @@
 falcon==1.1.0
 
+SQLAlchemy!=1.1.5,!=1.1.6,!=1.1.7,!=1.1.8,>=1.0.10 # MIT
```

Comment:
is this required? https://github.com/openstack/oslo.db/blob/4.24.1/requirements.txt#L11